x-pack/plugins/cases/server/client/cases/utils.ts

```diff
@@ -226,16 +233,46 @@ export const FIELD_INFORMATION = (
 };
 
 export const transformers: Record<string, Transformer> = {
-  informationCreated: ({ value, date, user, ...rest }: TransformerArgs): TransformerArgs => ({
-    value: `${value} ${FIELD_INFORMATION('create', date, user)}`,
+  informationCreated: ({
+    caseUrl,
+    date,
+    user,
+    value,
+    ...rest
+  }: TransformerArgs): TransformerArgs => ({
+    value: `${value} ${
+      caseUrl != null
+        ? `[${FIELD_INFORMATION('create', date, user)}](${caseUrl} )`
+        : FIELD_INFORMATION('create', date, user)
+    }`,
     ...rest,
   }),
-  informationUpdated: ({ value, date, user, ...rest }: TransformerArgs): TransformerArgs => ({
-    value: `${value} ${FIELD_INFORMATION('update', date, user)}`,
+  informationUpdated: ({
+    caseUrl,
+    date,
+    user,
+    value,
+    ...rest
+  }: TransformerArgs): TransformerArgs => ({
+    value: `${value} ${
+      caseUrl != null
+        ? `[${FIELD_INFORMATION('update', date, user)}](${caseUrl} )`
+        : FIELD_INFORMATION('update', date, user)
+    }`,
     ...rest,
   }),
-  informationAdded: ({ value, date, user, ...rest }: TransformerArgs): TransformerArgs => ({
-    value: `${value} ${FIELD_INFORMATION('add', date, user)}`,
+  informationAdded: ({
+    caseUrl,
+    date,
+    user,
+    value,
+    ...rest
+  }: TransformerArgs): TransformerArgs => ({
+    value: `${value} ${
+      caseUrl != null
+        ? `[${FIELD_INFORMATION('add', date, user)}](${caseUrl} )`
```

Comment:
I had to put an extra space or we got a `)` in the href (at least in jira). The space doesn't effect the nav but the `)` can have unexpected effects